core/domain/user_services.py

```diff
@@ -1546,6 +1556,14 @@ def update_dashboard_stats_log(user_id):
 
 
 def is_at_least_moderator(user_id):
+    """Checks if a user with given user_id is at least a moderator.
+
+    Args:
+        user_id: str. ID of user.
+
+    Returns:
+        Returns True if user is atleast a moderator otherwise returns False.
```

Comment:
Change to- 'bool. True if user is at least a moderator, False otherwise'. Always remember to write datatypes as well.